### DISCUSSION TITLE
Installing/upgrading klog in Ubuntu

### DISCUSSION THREAD

**Discussion by ghost** [ea4k/klog#603] [Q&A]
Hi, I am trying to upgrade my klog (KLog 1.8.6 By [EA4K](https://www.qrz.com/db/ea4k) - 2002-2021) who I first installed through the "sotware center" of my Ubuntu system. 

But after downloading the latest release and trying to install it using the command line "sudo dpkg -i klog_2.3.2-0raspbian1_armhf.deb"  I get an error: 
dpkg: fout bij verwerken van archief klog_2.3.1-0raspbian1_armhf.deb (--install):
 pakketarchitectuur (armhf) komt niet overeen met het systeem (amd64) << packet architecture does not match the system...>>
Fouten gevonden tijdens verwerken van:
klog_2.3.1-0raspbian1_armhf.deb

My sys: Ubuntu 22.04.2 LTS / Dell Inc. Inspiron 15 3511 /  16 GiB / 11th Gen Intel® Core™ i7-1165G7 @ 2.80GHz × 8 / 64 bit / Gnome vers 42.9 

ps. where can I find install.txt on github?

**Comment by ea4k** [comment]
The .deb that is in the repository is only for raspbian, not for Ubuntu.

You can find the instructions to compile here:

https://raw.githubusercontent.com/ea4k/klog/master/src/INSTALL-linux

**Reply by ghost** [reply]
Hi thanks for yr quick response. I tried to reach you via e-mail QRZ, I am not on telegram...

So it is not not possible to upgrade directly in Ubuntu (it's "software center" mentions klog not as open software??? while source code is shown here???)

Or to I have to compile/convert raspbian .deb vers to ubuntu first, then try to run the dpkg?
Sri I am new in Linux. I 'll look at your instructions.
ON5TTT 73's

Ps. On my WIN 11 (same LPT as mentioned abv.) i cant run it neither. After forced install (blocked first by defender...)  it gets stuck.
On WIN 10 (old LPT) all went ok.

**Comment by ghost** [comment]
Hi, I have installed QT5 but that does not help.
Command Untar (unrar??) klog is not executed.
I tried sudo dpkg -i klog_2.3.2.... (still same error)
I tried sudo apt install klog_2.3.2... and get (in the directory were it is listed "Donloads")
Packet is read
Tree is build
Status is read
Cann't find klog_2.3.2...
Help

Can anyone help how to isnstall klog in Ubuntu?
Thanks

**Comment by ghost** [comment]
In WIN10 with latest klog version I can not enter a qso on 10 MHZ, system crashes...

**Comment by ea4k** [comment]
> In WIN10 with latest klog version I can not enter a qso on 10 MHZ, system crashes... 

Added a bug report:
https://github.com/ea4k/klog/issues/604
